Documentation/README_HandCoach.md

```diff
@@ -0,0 +1,76 @@
+# Hand Coach
```

Comment:
```suggestion
# Hand coach
```

Reply:
not sure about this one - our guidelines would suggest this should be lower case if we're not aiming to make this a brand name
(this would apply to all other occurrences in this doc as well)

Reply:
ok I will go through and change

Reply:
going through and changing to Lower case. Is this the case for the asset files? for ex:  MixedRealityToolkit.SDK/Experimental/HandCoach

Reply:
no it's only for mentioning the feature in the docs - if you're directly using the class name HandCoach with a link to the api you can also stick to upper case, but just for describing the feature in the docs we should stick to lower case as it's not a brand or name

you can find guidance on doc styleguide here: https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#capitalization